app/controllers/wizard/steps/import_dates_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Wizard
+  module Steps
+    class ImportDatesController < BaseController
+      def index
+        @import_date = Wizard::Steps::ImportDate.new(session)
+      end
+
+      def create
+        @import_date = Wizard::Steps::ImportDate.new(session, permitted_params)
+
+        if @import_date.valid?
+          @import_date.save!
```

Comment:
Do we want the primadonna method, here? I think that effectively would propagate an error normally in the situation where the record was invalid and normally be handled by the base controller which would render any error messages

Reply:
Tbh Rubocop was crying about it. I'm not very keen on it.